### PR TITLE
Contributor credits and updated german translation

### PR DESCRIPTION
**Pull request by whot**
I got reminded again why I never switch any UI to german...

### CODE REVIEW

File: data/ui/AboutDialog.ui.in [hunk 1]
```diff
@@ -15,8 +15,21 @@
...
 
+    <!--  Contributor list: git log | git shortlog \-\-summary | cut -f2 -->
 Contributors:
-	Dan Callaghan &lt;djc@djc.id.au&gt;
+        AsavarTzeth
+        Benjamin Tissoires
+        Dan Callaghan
+        FFY00
+        Ikey Doherty
+        Jente Hidskes
```
Comment (Hjdskes) [inline data/ui/AboutDialog.ui.in]: I'm already under authors, should I be under contributors too?

File: data/ui/AboutDialog.ui.in [hunk 2]
```diff
@@ -15,8 +15,21 @@
...
+        Julian Richen
+        Peter Hutterer
```
Comment (Hjdskes) [inline data/ui/AboutDialog.ui.in]: Same here for you.

File: data/ui/AboutDialog.ui.in (hunk 1 above)
Comment (whot) [reply data/ui/AboutDialog.ui.in]: eh, it's just the git shortlog. Figured leaving it as-is is easier than thinking about it :)

Comment (Hjdskes) [reply data/ui/AboutDialog.ui.in]: Alrighty, let's merge this then!

**Pull request merged**